bin/ch-fromhost

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Note: The ${x:+y} construction yields $y if $x is set and not null, or $x
```

Comment:
This should say `${x:+$y}` instead.  The expansion of `${x:+y}` yields exactly `y`, not the value of `$y`, if `$x` has a non-empty value.  Also, if `$x` is empty or unset, **nothing** is substituted, not `$x`.  This is an important distinction for word expansion because "nothing" does not count as a word.